File: app/src/main/java/com/ajdi/yassin/popularmoviespart1/data/remote/api/MoviesResponse.java
package com.ajdi.yassin.popularmoviespart1.data.remote.api;

import com.ajdi.yassin.popularmoviespart1.data.remote.model.Movie;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Yassin Ajdi.
 */
public class MoviesResponse {

    @SerializedName("page")
    private int page;

    @SerializedName("total_results")
    private int totalResults;

    @SerializedName("total_pages")
    private int totalPages;

    @SerializedName("results")
    private List<Movie> movies;
}

File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/MoviesResponse.kt
package com.ajdi.yassin.popularmovies.data.local.model

import com.google.gson.annotations.SerializedName

/**
 * Created by Yassin Ajdi.
 */
data class MoviesResponse (

    @SerializedName("page")
    var page: Int = 0,

    @SerializedName("total_results")
    var totalResults: Int = 0,

    @SerializedName("total_pages")
    var totalPages: Int = 0,

    @SerializedName("results")
    var movies: List<Movie>? = null
)
